docs/quickstart.md

```diff
@@ -280,6 +283,24 @@ glm = nmo.glm.GLM(observation_model=nmo.observation_models.BernoulliObservations
 
 ```
 
+### **Counts with Supra-Poisson Variability**
+
+A [Negative Binomial observation model](nemos.observation_models.NegativeBinomialObservations) is appropriate for describing a counting process (such as neural spike counts) that exhibits supra-Poisson variability — that is, it is over-dispersed relative to a Poisson model (equivalently, the variance of the counts is larger than the mean).
+
+To set up a Negative Binomial GLM, provide this `observation_model` at initialization. You can regulate the degree of overdispersion by setting the `scale` parameter: the larger the `scale`, the more dispersed the Negative Binomial will be compared to a Poisson model.
+
+```{code-cell} ipython3
+
+import nemos as nmo
+
+# set up a Negative Binomial GLM for modeling counts (with scale = 1)
+# note: you should tune the scale parameter (for example, through cross-validation)
```

Comment:
this shouldn't be in a comment, but above or in an admonition (and link to something about cross-validation)